File: code/josh/HTML + CSS + FLASK/josh_lab05/unit_converter.py
#Unit Converter

def unit_converter():
    user_distance = int(input("What is the distance (ft, mi, m, km, yd or in)? "))
    units_input = input("What are the input units (ft, mi, m, km, yd, or in)? ")
    units_output = input("What are the output units (ft, mi, m, km, yd, or in)? ")
    conversion = {"ft": 0.3048, "mi": 1609.34, "m": 1, "km": 1000, "yd": 0.9144, "in": 0.0254}
    new_distance = user_distance * conversion[units_input]
    new_units = new_distance / conversion[units_output]
    print(f"{user_distance} {units_input} is {new_units} {units_output}.")

unit_converter()

File: code/lizzie/javascript/lab01/unit converter/unit-converter.js
const conversionDict = {
    ft: 0.3048,
    mi: 1609.34,
    m: 1,
    km: 1000,
    yd: 0.9144,
    in: 0.0254
}

const userDistance = parseFloat(prompt('What is the distance being converted?'));
const distance = parseFloat(userDistance);

const inputUnit = prompt('What are the input units?');

const outputUnit = prompt('What are the input units?');

const meterConversion = conversionDict[inputUnit] * distance
const conversion2 = meterConversion / conversionDict[outputUnit]

alert(`${distance} ${inputUnit} is ${conversion2.toFixed(3)} ${outputUnit}`)
